tasks/scripts.js

```diff
@@ -16,16 +17,6 @@ const scriptsDir = `${rootDir}/app/scripts`;
 
 function getWebpackConfig(testing) {
   return {
-    entry: testing ? null : {
-      background: `${scriptsDir}/background.js`,
-      contentscript: `${scriptsDir}/contentscript.js`,
-      options: `${scriptsDir}/options.js`,
-      popup: `${scriptsDir}/popup.js`,
-      sandbox: `${scriptsDir}/sandbox.js`,
-
-      vendor: ["jquery", "lodash"],
-      jquery: `${scriptsDir}/exporter/jquery.js`,
-    },
```

Comment:
We don't need `vendor` and `jquery` any more. So here we remove `entry` field and use `vinyl-named` to name files on the pipe.